src/entities/Snapshot/utils.ts

```diff
@@ -203,3 +204,8 @@ export function isSameAddress(userAddress?: string | null, address?: string | nu
     getChecksumAddress(userAddress) === getChecksumAddress(address)
   )
 }
+
+export async function getUsersWhoVoted(usersWithoutBadge: string[]) {
```

Comment:
Maybe `getUsersFromVotes`? And rename `usersWithoutBadge` to `users`? Or move this to Badge utils if we want to make it specific to badges.